src/index.js

```diff
@@ -54,30 +56,22 @@ app.get("/health", (_, res) => {
 });
 
 // Routes
-app.use("/auth", authRoute);
-app.use("/", blogRoute);
+app.use("/api/auth", authRoute);
+app.use("/api", blogRoute);
 app.use(notFoundMiddleware)
 app.use(errorHandler)
 
 const PORT = process.env.PORT || 3000;
 
-// Connect to the database
-mongoose
-  .connect(process.env.MONGODB_URL, {
-    useNewUrlParser: true,
-    useUnifiedTopology: true,
-  })
-  .then(() => {
-    console.log("Connected to the database!");
-  })
-  .catch((err) => {
-    console.log("Cannot connect to the database!", err);
-  });
-
 // Start the server
 app.listen(PORT, () => {
-  console.log(`Server is running on http://localhost:${PORT}`);
+  console.log(`> Server is running on http://localhost:${PORT}`);
   console.log(
-    `Swagger API documentation available at http://localhost:${PORT}/api-docs`
+    `> Swagger API documentation available at http://localhost:${PORT}/api-docs`
   );
 });
+
+// Connect database to app
+connectDatabase();
+// seed the database
+seedDB();
```

Comment:
Remove the `seedDB()` call from here and instead add a script in `package.json` 
```json
"db:seed": "node src/db/seed.js"
```

Reply:
ok